ibm_catalog.json

```diff
@@ -129,6 +129,20 @@
             {
               "key": "private_cert_engine_config_template_name"
             },
+            {
+              "key": "client_auth_methods",
+              "options": [
+                {
+                  "displayname": "username",
```

Comment:
wondering if we could have a better display name, like "IAM username"? Just by `username` its not very clear what username we are referring to.

Reply:
updated